frontend/templates/cp_main.html

```diff
@@ -161,7 +161,8 @@ <h2 class="card-title">
     $.getJSON("https://srhpyqt94yxb.statuspage.io/api/v2/incidents/unresolved.json", function(data) {
         if (data.incidents) {
             data.incidents.forEach(function(incident) {
-                addAlert("warning", "Discord's " + incident.status + " an incident: " + incident.name + ". Functionality may be limited.");
+                const incident_link = `<a href="${incident.shortlink}" target="_blank">${incident.name}</a>`
```

Comment:
Use camelCase for consistency with the rest of the file: `incidentLink`.

Reply:
Done in f18d968 .